aten/src/ATen/core/dynamic_type.cpp

```diff
@@ -365,7 +365,7 @@ DynamicType::Ptr IValue::TagType<c10::DynamicType>::get(const c10::IValue& v) {
 
 DynamicTypePtr ivalue::TupleTypeFactory<c10::DynamicType>::create(
     std::vector<TypePtr> elemTypes) {
-  return DynamicTypeFactory::create<TupleType>(std::move(elemTypes));
+  return DynamicTypeFactory::create<TupleType>(elemTypes);
```

Comment:
Hmm, create<> doesn't have a ctor? Interesting. TypePtr is not trivially copyable so shouldn't this be const ref?

Reply:
Will check in the next patch